lua/starfall/sflib.lua

```diff
@@ -230,6 +230,15 @@ SF.BurstObject = {
 			end
 			obj.val = new
 		end,
+		useNoThrow = function(self, ply, amount)
```

Comment:
Would've been simpler to just check and use separately but w/e. Although I would prefer it be named `tryUse` or something along the lines of that